tensorflow/core/BUILD

```diff
@@ -1349,60 +1349,63 @@ cc_library(
+               "//tensorflow/core/kernels:bincount_op",
+               "//tensorflow/core/kernels:boosted_trees_ops",
+               "//tensorflow/core/kernels:candidate_sampler_ops",
+               "//tensorflow/core/kernels:checkpoint_ops",
+               "//tensorflow/core/kernels:collective_ops",
+               "//tensorflow/core/kernels:control_flow_ops",
+               "//tensorflow/core/kernels:ctc_ops",
+               "//tensorflow/core/kernels:cudnn_rnn_kernels",
+               "//tensorflow/core/kernels:data_flow",
+               "//tensorflow/core/kernels:dataset_ops",
+               "//tensorflow/core/kernels:decode_proto_op",
+               "//tensorflow/core/kernels:encode_proto_op",
+               "//tensorflow/core/kernels:fake_quant_ops",
+               "//tensorflow/core/kernels:function_ops",
+               "//tensorflow/core/kernels:functional_ops",
+               "//tensorflow/core/kernels:grappler",
+               "//tensorflow/core/kernels:histogram_op",
+               "//tensorflow/core/kernels:image",
+               "//tensorflow/core/kernels:io",
+               "//tensorflow/core/kernels:linalg",
+               "//tensorflow/core/kernels:list_kernels",
+               "//tensorflow/core/kernels:lookup",
+               "//tensorflow/core/kernels:logging",
+               "//tensorflow/core/kernels:manip",
+               "//tensorflow/core/kernels:math",
+               "//tensorflow/core/kernels:multinomial_op",
+               "//tensorflow/core/kernels:nn",
+               "//tensorflow/core/kernels:parameterized_truncated_normal_op",
+               "//tensorflow/core/kernels:parsing",
+               "//tensorflow/core/kernels:partitioned_function_ops",
+               "//tensorflow/core/kernels:ragged_ops",
+               "//tensorflow/core/kernels:random_ops",
+               "//tensorflow/core/kernels:random_poisson_op",
+               "//tensorflow/core/kernels:remote_fused_graph_ops",
+               "//tensorflow/core/kernels:required",
+               "//tensorflow/core/kernels:resource_variable_ops",
+               "//tensorflow/core/kernels:rpc_op",
+               "//tensorflow/core/kernels:scoped_allocator_ops",
+               "//tensorflow/core/kernels:sdca_ops",
+               "//tensorflow/core/kernels:searchsorted_op",
+               "//tensorflow/core/kernels:set_kernels",
+               "//tensorflow/core/kernels:sparse",
+               "//tensorflow/core/kernels:state",
+               "//tensorflow/core/kernels:stateless_random_ops",
+               "//tensorflow/core/kernels:string",
+               "//tensorflow/core/kernels:summary_kernels",
+               "//tensorflow/core/kernels:training_ops",
+               "//tensorflow/core/kernels:word2vec_kernels",
+           ] + tf_additional_cloud_kernel_deps() +
+           select({
+               "//tensorflow:no_nccl_support": [],
+               "//tensorflow:with_cuda_support_windows_override": [],
```

Comment:
I agree, but the original was using a function which was in essence inserting a switch call with //tensorflow:with_cuda_support_windows_override, I didn't want to change the logic since I don't know whether `//tensorflow:with_cuda_support_windows_override==//tensorflow:windows` or has some extra requirements. If you want I can change it to `//tensorflow:windows`. Please let me know what you prefer.

Reply:
with_cuda_support_windows_override means cuda enables and windows.
But I think removing this dependency for all windows cases is the better approach.

Reply:
@gunan,

Done